org-cyf-itd/content/blocks/step-6/instructions/index.md

```diff
@@ -1,36 +1,55 @@
 
-Participants are required to score 100 or higher on the Duolingo English Practice Test, which corresponds to CEFR level B2. We set this level because we have learned from experience that participants without it are unable to follow the course materials in subsequent courses.
+This step is an opportunity to practice this with a practical task which is outside your current knowledge of CSS and HTML. We have provided a [CodePen](https://codepen.io/Ara225/pen/JoYbRVd) which contains some HTML code. Your task is to use CSS to style the HTML so that:
+- The element with the class of "parent" has a dotted border
+- The element with the class of "parent" is centered on the page even if the screen size changes
+- The div elements with the class "child" have background colors, borders, rounded corners and margins between them.
+- The div elements with the class "child" stay inside the element with the class parent even if the screen size changes
 
-**We want you to succeed.** If you have not yet reached this level, it is better to work on your English _first_, then apply to us when you are _ready_.
+You must not change the HTML in anyway.
 
+You may use any colors you like - they aren't the important bit in this task.
 
-### What Should You Do?
+Here are some visual examples:
+
+#### On a large screen
+![alt text](animals-larger-screen.png)
+#### On a much smaller screen
+![alt text](animals-smaller-screen.png)
 
-##### 1. Preparation
+Assuming you were new to HTML and CSS when you started this course, this task is outside your current knowledge. This makes it an excellent candidate for AI assistance, as you can use AI to help you understand the new concepts.
+
+### What Should You Do?
 
-1. Find 50 minutes where you will not be interrupted.
-2. [Register on the Duolingo website](https://englishtest.duolingo.com/register) to do the Practice Test.
-3. You need to look at your camera during some exercises, so test that your camera and microphone are working. You may need to do this test on your phone.
+#### 1 - Fork the CodePen
+Fork the [exercise CodePen](https://codepen.io/Ara225/pen/JoYbRVd) into your CodePen account
 
-##### 2. Take the test
+#### 2 - Make a Plan 
+Start by thinking about what you need to know to solve this problem. What might be a good first question to ask the AI? How can you avoid having the AI do your thinking for you?
 
-1. Go to the [Duolingo Practice Test](https://englishtest.duolingo.com/test/practice) page.
-2. Take the test, which will be around 45 minutes.
-3. Take a screenshot of your result.
+#### 3 - Interact with the AI
+Use an AI tool to help you solve the problem. This can mean asking it to explain concepts, provide examples and explain code etc. It must not mean copying code from the AI or blindly following its advice without understanding why. You may wish to look at step 2 again and/or search resources for using AI appropriately. If you're stuck, Google or ask in the #step-6-questions-support channel on Slack.
```

Comment:
```suggestion
Use an AI tool to help you solve the problem. This can mean asking it to explain concepts, provide examples, explain code, etc. It must not mean copying code from the AI or blindly following its advice without understanding why. You may wish to look at step 2 again and/or search resources for using AI appropriately. If you're stuck, Google or ask in the #step-6-questions-support channel on Slack.
```